share/instruments/instruments.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Note for myself. Or @danieljray if you manage to do it : Change the commit message and keep only the second line.

Reply:
Also fix the indent for Genre